packages/theming/atlas/src/themesource/atlas_core/web/layouts/_layout-atlas-responsive.scss

```diff
@@ -127,6 +127,12 @@
         // Topbar
         .region-topbar {
             padding: 0 $spacing-small;
+
+            .toggle-btn {
+                border-color: transparent;
```

Comment:
no `padding:0` just like the other?

Reply:
It takes from the previous file (_layout_atlas)

Reply:
Fixed